.github/workflows/build.yml

```diff
@@ -33,28 +33,29 @@ env:
   BINARYEN_VERSION: version_111
 
 jobs:
-  matrix:
+  dynamic-profiles:
     runs-on: ubuntu-latest
     outputs:
-      matrix: ${{ steps.set-matrix.outputs.matrix }}
+      profiles: ${{ steps.set-matrix.outputs.profiles }}
     steps:
       - id: set-matrix
         run: |
-          variants='{"profile": "debug", "profile_flags": ""}'
+          profiles='{"name": "debug", "flags": ""}'
           if [ "${{ inputs.release }}" = "true" ]; then
-            variants+=', {"profile": "release", "profile_flags": "--release"}'
+            profiles+=', {"name": "release", "flags": "--release"}'
           fi
-          matrix="{\"include\": [${variants}]}"
-          echo "Matrix: ${matrix}"
-          echo "matrix=${matrix}" >> $GITHUB_OUTPUT
+          profiles="[${profiles}]"
+          echo "Profiles: ${profiles}"
+          echo "profiles=${profiles}" >> $GITHUB_OUTPUT
 
   linux:
     runs-on: [ kuberunner, github-runner-02 ]
-    needs: matrix
+    needs: dynamic-profiles
```

Comment:
need to update https://github.com/gear-tech/gear/blob/master/.github/actions/label/build.js#L132 as well